code/datums/components/tackle.dm

```diff
@@ -156,9 +156,10 @@
 	var/mob/living/carbon/target = hit
 	var/mob/living/carbon/human/T = target
 	var/mob/living/carbon/human/S = user
-	var/tackle_word = isfelinid(user) ? "pounce" : "tackle" ///If cat, "pounce" instead of "tackle".
 
 	var/roll = rollTackle(target)
+	var/tackle_word = isfelinid(user) || iskepori(user) ? "pounce" : "tackle" ///If cat, "pounce" instead of "tackle".
```

Comment:
parentheses here would make it less confusing, also the comment could use an update, same with the leap_word change above.